tests/test_timing_properties.py

```diff
@@ -0,0 +1,143 @@
+"Contains a collection of pytest tests that validates the timing properties."
+
+import random
+
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+from nidaqmx import DaqError
+from nidaqmx.constants import AcquisitionType, SampleTimingType
+
+
+class TestTimingProperty(object):
+    """Contains a collection of pytest tests that validates the timing properties."""
```

Comment:
Remove test class and write test functions.

Reply:
Resolved.